src/placeos-models/user.cr

```diff
@@ -12,11 +12,37 @@ module PlaceOS::Model
   class User < ModelBase
     include RethinkORM::Timestamps
 
+    struct Email
```

Comment:
This could be better scoped beneath `PlaceOS::Model` directly, in a separate file